hrm/department_app/models/department.py

```diff
@@ -0,0 +1,14 @@
+"""Django models"""
+from django.db import models
+from djmoney.models.fields import MoneyField
+
+
+class Department(models.Model):
+    """Department"""
+    department = models.CharField('Department',max_length=50)
```

Comment:
Please could you replace the field **department** with **name**: **department.name** looks better than **department.department**.

Reply:
Fixed.